docs/code-snippets/outlook-snippets.yaml

```diff
@@ -672,6 +672,62 @@ Office.InfobarClickedEventArgs:interface:
         console.log(`Notification type: ${infobarDetails.infobarType}`);
         console.log(`Action type: ${infobarDetails.actionType}`);
     }
+Office.LoadedMessageCompose:interface:
+  - |-
+    // Gets the sender's email address of each selected message.
+    async function getSenderEmailAddress(item) {
+        const itemId = item.itemId;
+        await new Promise((resolve) => {
+            Office.context.mailbox.loadItemByIdAsync(itemId, (result) => {
+                if (result.status === Office.AsyncResultStatus.Failed) {
+                    console.log(result.error.message);
+                    return;
+                }
+
+                const loadedItem = result.value;
+                const sender = loadedItem.from.emailAddress;
+                appendToListItem(sender);
```

Comment:
is appendToListItem something that can/should be included as a supporting function?

Reply:
Good point. I changed this line of code, so that we wouldn't need to add the supporting function. This way, focus would remain on the loadItemByIdAsync API.